### PR TITLE
Upl links update

### PR DESCRIPTION
**Pull request by yipingliao**
@2is10 cc @eishay 
- Add microdata to owner pictures in user profile library cards
- Add links to follower pictures in user profile library cards (no microdata for these yet since followers are not shown for logged-out users; question noted for product - we can add microdata when we decide to show followers)

### CODE REVIEW

**Comment by eishay** [comment]
Great!
Do we have microdata link refs on user faces on library card?

File: kifi-backend/modules/shoebox/angular/src/userProfile/userProfile.js [hunk 1]
```diff
@@ -179,6 +179,10 @@ angular.module('kifi')
       return hasMoreLibraries;
     };
 
+    $scope.getUserProfileUrl = function (user) {
+      return userService.getProfileUrl(user.username);
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/userProfile/userProfile.js:183]: What would you think of having `routeService` be responsible for all formatting of our URLs?

File: kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html [hunk 2]
```diff
@@ -18,10 +18,15 @@
       </div>
       <div class="kf-upl-card-byline"><span class="kf-upl-card-by">by</span> {{lib.owner.firstName}} {{lib.owner.lastName}}</div>
     </a>
-    <a class="kf-upl-card-owner-pic" href="/{{lib.owner.username}}" ng-style="{'background-image': 'url(' + lib.ownerPicUrl + ')'}"></a>
+    <div itemscope itemtype="http://data-vocabulary.org/Person">
+      <a href="/{{lib.owner.username}}" itemprop="url">
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html:22]: should this be `getUserProfileUrl(lib.owner)`?

**Comment by 2is10** [comment]
:+1:

File: kifi-backend/modules/shoebox/angular/src/userProfile/userProfile.js (hunk 1 above)
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/userProfile/userProfile.js:183]: I like it. I will refactor.

File: kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html (hunk 2 above)
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/userProfile/userProfileLibrariesList.tpl.html:22]: Yes! Will fix.

**Pull request merged**